portal-frontend/src/app/features/applications/edit-submission/primary-contact/primary-contact.component.html

```diff
@@ -24,37 +23,41 @@ <h4>Primary Contact Information</h4>
         </p>
         <mat-button-toggle-group
           class="input"
-          [formControl]="isExistingOwner"
-          (change)="onSelectPrimaryContactType(isExistingOwner.value)"
+          [value]="!selectedThirdPartyAgent"
+          (change)="onSelectPrimaryContactType($event)"
         >
           <mat-button-toggle [value]="true">Yes</mat-button-toggle>
           <mat-button-toggle [value]="false">No</mat-button-toggle>
         </mat-button-toggle-group>
       </div>
 
       <!-- Owner -->
-      <div *ngIf="selectedThirdPartyAgent === false && parcelOwners.length > 0">
-        <label *ngIf="parcelOwners.length"
+      <div *ngIf="selectedThirdPartyAgent === false">
+        <label *ngIf="selectedThirdPartyAgent === false && parcelOwners.length > 0"
           >Select a Primary Contact from the listed land owners
           <ng-container *ngIf="hasCrownParcels">and government contacts</ng-container></label
         >
 
-        <mat-radio-group color="primary" [value]="selectedOwnerUuid" required>
-          <mat-radio-button
-            *ngFor="let owner of parcelOwners"
-            [value]="owner.uuid"
-            class="radio-option"
-            (click)="onSelectOwner(owner.uuid)"
-            required
-          >
+        <mat-radio-group
+          [formControl]="ownersList"
```

Comment:
You shouldn't have both a formControl and a value, as formControls also have their own value.

Reply:
Went with a getter/setter for this.

Rationale:
- any existing use of `selectedOwnerUuid` can stay as is
- `[value]` binding can be removed from template without further modification

Reply:
Looks good!